routers/userRouter.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+const { userController }  = require('../controllers');
+
+router.get('/signIn', userController.signIn);
```

Comment:
기획상 소셜 로그인 만을 구현하긴 하지만 확장성이 고려된 endpoint 작성을 지향해주세요
```suggestion
router.get('/kakao-signin', userController.signIn);
```

Reply:
수정했습니다